src/app/pages/subjects/subjects.js

```diff
@@ -0,0 +1,149 @@
+import BaseView from '~/helpers/backbone/view';
+import BaseModel from '~/helpers/backbone/model';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './subjects.hbs';
+import FilterButton from './filter-button/filter-button';
+import CategorySection from './category-section/category-section';
+
+function makeBook(title, categories) {
+    return {
+        title,
+        categories,
+        coverUrl: `https://placeholdit.imgix.net/~text?txtsize=33&txt=${title}&w=200&h=180`
+    };
+}
+
+@props({
+    template: template,
+    regions: {
+        filterButtons: '.filter-buttons',
+        bookViewer: '.book-viewer'
+    }
+})
+export default class Subjects extends BaseView {
+
+    constructor() {
+        super();
+        this.model = new BaseModel({viewIsReady: false});
```

Comment:
Now that I'm actually fetching data, I've got a better notion of how to work with Backbone models. `viewIsReady` will go away with my subjects-fetch branch, which will follow the branch I've made to fix up all these issues.